### PR TITLE
fixed the deprecated keywords in the neo4j queries

### LINKED ISSUE #154: Replace use of `id()` with `elementId()`
The [`id()`](https://neo4j.com/docs/cypher-manual/current/functions/scalar/#functions-id) function we use in our [main module](https://github.com/InternetHealthReport/internet-yellow-pages/blob/280243d10cc291cb7ac1241c914c40d2e16b46c8/iyp/__init__.py) is deprecated and should be replaced with [`elementId()`](https://neo4j.com/docs/cypher-manual/current/functions/scalar/#functions-elementid). I believe it's just a matter of replacing the function, but should be confirmed.

Newer versions of the neo4j Python module will raise a lot of warnings so we should address this at some point.

### PR DESCRIPTION
**Pull request by ronibhakta1**
@m-appel check this out i have made the asked changes...
closes #154 

## Description
id() changed with elementId() in neo4j with new required documentation [elementId()](https://neo4j.com/docs/cypher-manual/current/functions/scalar/#functions-elementid)

## How Has This Been Tested?

pre-commit checks 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: iyp/__init__.py [hunk 1]
```diff
@@ -747,16 +761,19 @@ def __init__(self, dir: str, prefix: str) -> None:
...
     def cached_object_exists(self, object_name: str) -> bool:
-        cache_file = f'{self.cache_file_prefix}{object_name}{self.cache_file_suffix}'
+        cache_file = f'{self.cache_file_prefix}{
+            object_name}{self.cache_file_suffix}'
         return os.path.exists(cache_file)
 
     def load_cached_object(self, object_name: str):
-        cache_file = f'{self.cache_file_prefix}{object_name}{self.cache_file_suffix}'
+        cache_file = f'{self.cache_file_prefix}{
+            object_name}{self.cache_file_suffix}'
```
Comment (m-appel) [inline iyp/__init__.py:770]: This line is broken now and causes a syntax error.

**Comment by m-appel** [comment]
Hi, I'm not sure what kind of code formatter you use, but please don't introduce unnecessary formatting changes as it crowds the PR. Our line length (specified in .flake8) is 120 characters.
The formatter also broke one line, which is why the pre-commit checks are failing.

So please only change the requested fields and *test* the functions. The testing part is the reason this is an issue and not just a simple Search + Replace, since it requires running a neo4j instance and confirming that the functions still behave the same.

**Comment by ronibhakta1** [comment]
> Hi, I'm not sure what kind of code formatter you use, but please don't introduce unnecessary formatting changes as it crowds the PR. Our line length (specified in .flake8) is 120 characters.
> The formatter also broke one line, which is why the pre-commit checks are failing.
> 
> So please only change the requested fields and *test* the functions. The testing part is the reason this is an issue and not just a simple Search + Replace, since it requires running a neo4j instance and confirming that the functions still behave the same.

I will take a look at it, I'm trying to understand the whole product itself.

**Comment by m-appel** [comment]
Hey, since this has become somewhat urgent, I am testing this myself.
In general, to get a database up and running, you should look at the main README.
To test the behavior, I am just running versions of the old and new queries against the webinterface and see if they behave the same.

**Comment by ronibhakta1** [comment]
> ,

Hey thanks a lot, I'm new to open source and contributing to many organisations. And due to different projects it's taking time to understand their projects. I have checked the issue I was facing on my end was due to vs code idk why it was making a mess on my Mac. Anyways thanks again I will get things done next time :)

**Comment by m-appel** [comment]
No worries, I still welcome you to try and getting IYP up and running and try it out.
I would have given you more time, but we just updated our neo4j version and are now getting spammed with "id() is deprecated" messages, so I want to merge this asap :)

**Comment by ronibhakta1** [comment]
I hope that I have made the changes as soon as possible, some IDE problems
but hope I made it in your comfortable time 😄

**Pull request merged**